builtin/credential/chef-node/path_config.go

```diff
@@ -0,0 +1,75 @@
+package chefNode
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func pathConfig(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: "config",
```

Comment:
This could be ambiguous.  While the backend doesn't currently support binding roles to an individual chef client, because I don't think it is a particularly good idea, I could see someone wanting this.  If the other binding endpoints are moved under /config I would think /config/client would be the logical endpoint to use for that.